ftw/contacts/tests/test_hide_contact_images.py

```diff
@@ -0,0 +1,45 @@
+from ftw.builder import Builder
+from ftw.builder import create
+from ftw.contacts.testing import FTW_CONTACTS_FUNCTIONAL_TESTING
+from ftw.testbrowser import browsing
+from plone.app.testing import TEST_USER_ID
+from plone.app.testing import setRoles
+from unittest2 import TestCase
+
+
+class TestHideContactImages(TestCase):
+
+    layer = FTW_CONTACTS_FUNCTIONAL_TESTING
+
+    def setUp(self):
+        self.portal = self.layer['portal']
+        setRoles(self.portal, TEST_USER_ID, ['Manager'])
```

Comment:
Use `self.grant('Manager') if you inherit from `ftw.contacts.tests.FunctionalTestCase`.